tensorflow/training/docker/1.15.4/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,160 @@
+FROM ubuntu:18.04
```

Comment:
@Teng-xu is this going to be the same DLC major version as the 1.15.3 containers? If so, can you make this change in place in the "1.15" directory?

Reply:
Updated